odk/extension/src/main/java/io/samagra/odk/collect/extension/interactors/ODKInteractor.kt

```diff
@@ -25,12 +23,4 @@ interface ODKInteractor {
     /** Resets everything in odk and deletes all data. */
     fun resetODK(listener: ODKProcessListener)
 
-    /** Opens the latest version related to the formId. Deletes any
-     *  saved instance of a form with this particular formId. */
-    fun openForm(formId: String, context: Context)
-
-    /** Opens a saved form. If no saved instance is found, opens a new form. */
-    fun openSavedForm(formId: String, context: Context)
-
-    fun prefillAndOpenForm(formId: String, tagValueMap: HashMap<String, String>, context: Context)
```

Comment:
Could you please also add a 1-2 line of documentation to describe this method?

Reply:
Added comments to `prefillAndOpenForm()` method